week4_Isotopes.Rmd

```diff
@@ -0,0 +1,117 @@
+
+# Prerequisites
+
+1. Have an understanding of what **isotopes** are and how to express them in different notations.
+
+2. Basic understanding of **R** programming, flow control, graphing, and tibble manipulation.
+
+# Background
+
+Because we just finished our lesson on what isotopes are and their notation *cough*, it is time to talk about **isotope fractionation**. **Isotope fractionation** is the separation or partitioning of a reservoir(s) stable isotopes due to a processe that use masses differently. Some processes depend on the relative differences in the masses of different isotopes in chemical equilibrium (**equilibrium isotopic fractionation**). Other processes depend on rate-dependent transfers or site-specific chemical binding effects (**kinetic isotopic fractionation**).
+
+Today we are going to talk about the evaporation of water which happens to be an example of a rate-dependent kinetic fractionation. Well... its kinetic in very dry air. Let's not go there for now. The fractionation from a process is described by using an often empirically derived non-dimensional parameter called a **fractionation factor**. Evaporation is considered to be a type of fractionation called a **Rayleigh distillation**. In an **open system**, isotopes will not return to the original reservoir once evaporated. At this point, we can write out a general form for a Rayleigh distillation that will tell us the isotopic ratio of the reservoir as a function of the remaining amount that hasn't been evaporated.
+
+$$
+\frac{R}{R_{0}} = f^{\alpha-1}
+$$
+
+where *R* = ratio of the isotopes (e.g., 18O/16O) left in the reservoir, *Ro* = initial ratio, *f* = fraction of material remaining in the reservoir, and *α* is the fractionation factor.
+
+We can do some algebra now and rewrite this in $\delta$ notation. I will spare you the trouble.
+
+$$
+ \\
+\delta_{l} = (\delta_{l,0} + 1000)f^{\alpha-1} - 1000
+$$
+
+where *0* stands for the initial δ-value of the liquid (*l*) for the case when α is the fractionation factor between vapor and liquid αvapor-liquid, and f is the fraction of liquid remaining.
+
+# Example
+
+The empirically derived fractionation factor associated with the phase transition during evaporation, vapor-liquid = 0.990 at 25°C. Think of this as saying that the new vapor is 10‰ lighter than the residual water. Now let's use the above equation to calculate the curve for a starting water delta value of a series of $\delta$ values from $\delta = -10$ to $\delta = -30$ permil.
+
+```{r, message=FALSE}
+library(dplyr)
+
+# session info
+starting_delta_values = seq(-10,30,10) #Starting delta valuyes for the resevoir
+alpha = 0.990 #An example fractionation factor
+
+remaining_Delta <- function(starting_delta_values, remaining_Water_Fraction, alpha) #Function which calculates the isotope value for the remaining water post-process
+{
+  remaining_Delta = (starting_delta_values +   1000)*(remaining_Water_Fraction^(alpha-1))-1000
```

Comment:
I know it can be kind of tedious (and don't feel like you have to do this by tomorrow), but in the future adding descriptive comments for EVERY single line of code is super helpful for students (and me lol).